### ISSUE TITLE
Recover carelessly closed upstream pull requests

### ISSUE THREAD

**Issue by aminya** [atom-community/atom#379]
At some point, the upstream leadership closed a bunch of pull requests just because they didn't care, and they just wanted to make the repo look "maintained". Now, that they have shut down the repo, we know that was just a sugarcoat on the status of the upstream repository and its leadership.

Here are a series of them I made, and they closed it without providing a good explanation.
https://github.com/atom/atom/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+author%3Aaminya

If you see the closed tab, you see a lot of similar pull requests. 
https://github.com/atom/atom/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed

**Comment by kaosine** [comment]
Yeah they definitely didn't have the team to maintain it properly nor did they care. I suggested trying to figure out what it'd take to get to get any of the ones over we could that might actually help fix things. If I had more know how to decipher most of it I would definitely help with that.
